### ISSUE TITLE
Some variables are not renamed in "Rename Symbol" functionality.

### ISSUE THREAD

**Issue by tomopy03** [microsoft/pyright#221]
**Describe the bug**
Some variables are not renamed in "Rename Symbol" functionality.

**Screenshots or Code**
![スクリーンショット 2019-08-26 11 27 21](https://user-images.githubusercontent.com/16932067/63661234-a246b000-c7f4-11e9-9cd5-fce0d516d833.png)

![スクリーンショット 2019-08-26 11 27 27](https://user-images.githubusercontent.com/16932067/63661239-a8d52780-c7f4-11e9-8c75-31159d84b20e.png)

**Expected behavior**
`cnt += 1`
After rename symbol
`count += 1`

**VS Code extension or command-line**
VS Code 1.37.1
Pyright 1.0.57

**Comment by giyyapan** [comment]
**Additional context**

Using `find all references` functionality shows the same result as the issue description
![image](https://user-images.githubusercontent.com/3479453/63664696-4f6ff700-c7fa-11e9-89c2-eb581a790bf5.png)

But it's current if triggered from this cursor location:
![image](https://user-images.githubusercontent.com/3479453/63664712-6f071f80-c7fa-11e9-96c1-af0e7c75ffcc.png)

I think it's might be related to the behavior of `+=` operator.

codes:
```python
def rename_test():
    cnt = 1
    while True:
        cnt += 1
        if cnt > 5:
            break
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This bug is now fixed in 1.0.58, which I just published.
